squadxml/squads.yaml

```diff
@@ -33,27 +34,38 @@ squads:
     web: www.folkarps.com
     picture: armogus_512.paa
     title: Folk ARPS
+    allMembers: true
   ch3:
     name: Folk ARPS
     nick: FA
     email: hosts@folkarps.com
     web: www.folkarps.com
     picture: ch3_512.paa
     title: Folk ARPS
+    allMembers: true
   coldwar:
     name: Folk ARPS
     nick: FA
     email: hosts@folkarps.com
     web: www.folkarps.com
     picture: coldwar_512.paa
     title: Folk ARPS
+    allMembers: true
   commend:
     name: Folk ARPS (Commendation)
     nick: FA
     email: hosts@folkarps.com
     web: www.folkarps.com
     picture: commend_512.paa
     title: Folk ARPS
```

Comment:
Guess we are keeping this one (commend) as "at hosts discretion"?

Reply:
Yes, that seemed reasonable to me.